source/includes/extracts-agg-operators.yaml

```diff
@@ -489,6 +489,11 @@ content: |
       * - Name
         - Description
 
+      * - :expression:`$rand`
+        - Returns a random float between 0 and 1
+        
+          .. versionadded:: 4.4.2
+
```

Comment:
Can you add this same `versionadded` to the `$sampleRate` entry just beneath as well please? Looks like I missed this in the review for that backport.